client/routes/router.js

```diff
@@ -28,12 +28,16 @@ FlowRouter.route('/', {
 		Tracker.autorun(function(c) {
 			if (FlowRouter.subsReady() === true) {
 				Meteor.defer(function() {
-					if (Meteor.user() && Meteor.user().defaultRoom) {
-						const room = Meteor.user().defaultRoom.split('/');
+					const user = Meteor.user();
+					if (user && user.defaultRoom) {
+						const room = user.defaultRoom.split('/');
 						FlowRouter.go(room[0], { name: room[1] }, FlowRouter.current().queryParams);
-					} else {
+					} else if (user && user.lastVisitedRoom) {
+						const room = user.lastVisitedRoom;
```

Comment:
Why user.lastVisitedRoom not working. I will slove this as hurry as I can(2 days). I need your help. can you please mark this with label progress?